taqueria-plugin-flextesa/package.json

```diff
@@ -38,6 +38,7 @@
 	},
 	"dependencies": {
 		"@taqueria/node-sdk": "^0.12.0",
+		"@taqueria/protocol": "^0.12.0",
```

Comment:
Should these not be referencing `^0.14.0`?

Reply:
Now I rebased on main and all are `0.14.4`